templates/searchv2/search.html

```diff
@@ -51,7 +51,11 @@
         {% if query %}
             {% include "misc/paginator.html" with position="top" %}
 
-            {% if object_list %}
+            {% if object_list == '*' %}
+                <p>
+                    Recherche invalide.
```

Comment:
```suggestion
                    {% trans "Recherche invalide." %}
```
Et il faut aussi traduire la chaîne dans le test correspondant.